container/src/lib.rs

```diff
@@ -102,7 +90,7 @@ pub trait PushInto<T> {
 /// decide to represent a push order for `extract` and `finish`, or not.
 pub trait ContainerBuilder: Default + 'static {
     /// The container type we're building.
-    type Container: Container + Clone + 'static;
+    type Container: WithProgress + Default + Clone + 'static;
```

Comment:
`WithProgress : Default`, so is the `+ Default` redundant, or perhaps helpful because that `WithProgress` constraint isn't locked down?

Reply:
Removed, was redundant.